src/main/scala/chisel3/tester/internal/Testers2.scala

```diff
@@ -2,16 +2,26 @@
 
 package chisel3.tester.internal
 
-import scala.util.DynamicVariable
 import chisel3.experimental.MultiIOModule
+import firrtl.AnnotationSeq
+import firrtl.stage.phases.DriverCompatibility.TopNameAnnotation
+import treadle.WriteVcdAnnotation
 
 import scala.util.DynamicVariable
 
 // Internal common set of options to be understood by all backends
+@deprecated("Use annotation based options instead. See: .withAnnotations", "chisel-testers2 20190604")
```

Comment:
May also want to note that the write vcd annotation probably won't work with a theoretical Verilator backend since it seems to be a treadle specific thing.

Reply:
Added this as //TODO